Yoast/wordpress-seo
# Bump plugin-grunt-tasks, drops sass dependency

## Context


* Removes the sass dependecy we had.

## Test instructions

This PR can be tested by following these steps:

* Pull, run `yarn`, check that all grunt tasks still work.

## Quality assurance
